tests/test_AllocateSizeMoreThanClusterSize.py

```diff
@@ -0,0 +1,332 @@
+"""
+Automates the following test
+
+OCS-269 - FT-OCP-Create-PV-AllocateSizeMoreThanClusterSize
+Verify a PVC creation by allocating more storage
+than what is available in Ceph
+"""
+
+import logging
+import yaml
+import os
+import pytest
+import ocs.defaults as defaults
+
+from ocs import ocp
+from utility import utils, templating
+from utility.utils import run_cmd
+from ocsci.testlib import tier1, ManageTest
+
+log = logging.getLogger(__name__)
+
+PVC_YAML = os.path.join(
+    "templates", "RBD_PersistentVolumeClaim.yaml"
+)
+POD_YAML = os.path.join(
+    "templates", "RBD_NginxPod.yaml"
+)
+SC_YAML = os.path.join(
+    "templates", "RBD_StorageClass.yaml"
+)
+
+
+TEMP_PVC_YAML_FILE = '/tmp/pvc_test.yaml'
```

Comment:
I suggest to use `tmpdir` builtin fixture creating temporary files. Example in https://github.com/red-hat-storage/ocs-ci/pull/98/files#diff-484462fced51d1a06b1d93b4a44dd535R31

Reply:
This is no longer relevant as temporary yaml files are handled by `ocs-ci` classes.